apps/web/pages/[teamSlug]/team/[userId].page.tsx

```diff
@@ -32,6 +38,17 @@ const UserDetailsPage = (): JSX.Element => {
   const isAdmin = meData?.user.role === Role.Admin
   const [, createProjectMembership] = useProjectMembershipCreateMutation()
   const [, deleteProjectMembership] = useProjectMembershipDeleteMutation()
+  const [, updateUserCapacity] = useUserCapacityUpdateMutation()
+
+  const submitHandler = async (data: UserDetailsForm) => {
+    const response = await updateUserCapacity({
+      availableMinutesPerWeek: +data.availableMinutesPerWeek,
```

Comment:
An empty string (`""`) for `availableMinutesPerWeek` will be parsed to `0`.
But it should be `null` instead.

Or am I wrong?